src/components/DesignTheme/View/__tests__/PersonaView.test.jsx

```diff
@@ -0,0 +1,528 @@
+        <PersonaView content={mock_mandatory} />
+      </MemoryRouter>
+    </Provider>,
+  );
+  // title
+  expect(getByText(/Aguzzoli Claudia Dana/i)).toBeInTheDocument();
+
+  // role
+  expect(getByText(/Consigliere comunale/i)).toBeInTheDocument();
+
+  // tipologia persona
+  expect(getByText(/Tipologia di persona: Politica/i)).toBeInTheDocument();
+
+  // organizzazione di riferimento
+  const organizzazione_riferimento = await waitForElement(() =>
+    getByText(/SIET/i),
+  );
+  expect(organizzazione_riferimento).toBeInTheDocument();
+
+  // collegamenti_organizzazione_l1
+  const collegamenti_organizzazione_l1 = await waitForElement(() =>
+    getByText(/Unità organizzativa di primo livello/i),
+  );
+  expect(collegamenti_organizzazione_l1).toBeInTheDocument();
+
+  // collegamenti_organizzazione_l2
+  const collegamenti_organizzazione_l2 = await waitForElement(() =>
+    getByText(/Unità organizzativa di secondo livello/i),
+  );
+  expect(collegamenti_organizzazione_l2).toBeInTheDocument();
+});
+
+test('Checks all field when we have filled up mock', async () => {
+  const { getByText, getByAltText } = render(
+    <Provider store={store}>
+      <MemoryRouter>
+        <PersonaView content={mock_allfields} />
+      </MemoryRouter>
+    </Provider>,
+  );
+
+  // atto_nomina
+  expect(getByText(/Atto di nomina/i)).toBeInTheDocument();
+
+  // biografia
+  expect(getByText(/Ruolo\/Biografia/i)).toBeInTheDocument();
+
+  // competenze
+  expect(getByText(/Competenze/i)).toBeInTheDocument();
+
+  // curriculum_vitae
+  expect(getByText(/Curriculum vitae/i)).toBeInTheDocument();
+
+  // data_insediamento
+  expect(getByText(/Data di insediamento/i)).toBeInTheDocument();
+  expect(getByText(/12-03-2020/i)).toBeInTheDocument();
+
+  // deleghe
+  expect(getByText(/Deleghe/i)).toBeInTheDocument();
+
+  // description
+  expect(getByText(/Lorem ipsum description/i)).toBeInTheDocument();
+
+  // contatti: email, telefono, Contatti, informazioni_di_contatto
+  expect(getByText(/Contatti/i)).toBeInTheDocument();
+  expect(getByText(/lucabel@redturtle.it/i)).toBeInTheDocument();
+  expect(getByText(/3459988767/i)).toBeInTheDocument();
+  expect(getByText(/Altre informazioni di contatto/i)).toBeInTheDocument();
+
+  // foto_persona
+  expect(getByAltText(/Aguzzoli Claudia Dana/i)).toBeInTheDocument();
+
+  // responsabile_di
+  expect(getByText(/Responsabile di/i)).toBeInTheDocument();
+
+  // ulteriori_informazioni
+  expect(getByText(/Ulteriori informazioni text/i)).toBeInTheDocument();
+});
+
+test('Specific fields not in page if data_conclusione_incarico compiled', async () => {
+  const { getByText, queryByText } = render(
+    <Provider store={store}>
+      <MemoryRouter>
+        <PersonaView content={mock_allfields_and_fine_rapporto} />
+      </MemoryRouter>
+    </Provider>,
+  );
+
+  // Mi pare di capire che getByText se il testo non c'è, dia errore.
+  // queryByText l'ho testato verificando che con not.toBeNull da errore
```

Comment:
Corretto, quando il testo DEVE esserci usi `getByText` che te lo controlla in automatico, altrimenti `query` è meglio di `get`.